1-exercises/B-setting-values/exercise1.js

```diff
@@ -23,6 +23,11 @@ let capitalCities = {
 */
 
 // write code here
+capitalCities.UnitedKingdom.population = 8980000;
+capitalCities.China.population = 21500000;
+capitalCities.Peru = {};
+capitalCities.Peru.name = "Lima";
+capitalCities.Peru.population = 975000;
```

Comment:
this works, you can also set the properties of the Peru object all at once, by setting the name & population when you create the object